src/main/java/me/zeroeightsix/kami/module/modules/render/Nametags.kt

```diff
@@ -0,0 +1,529 @@
+                val color = healthColorGradient.get(duraPercentage).toHex()
+                val text = duraPercentage.roundToInt().toString()
+                val textWidth = mc.fontRenderer.getStringWidth(text)
+                mc.fontRenderer.drawString(text, -round(textWidth / 2f), 0f, color, textShadow.value)
+                glPopMatrix()
+            }
+
+            if (count.value && itemStack.count > 1) {
+                val itemCount = itemStack.count.toString()
+                mc.fontRenderer.drawStringWithShadow(itemCount, 17f - mc.fontRenderer.getStringWidth(itemCount), 9f, 0xFFFFFF)
+            }
+
+            glTranslatef(0f, -2f, 0f)
+            if (enchantment.value) enchantmentText.draw(scale = 0.5f, lineSpace = 3, verticalAlign = TextComponent.VAlign.BOTTOM)
+
+            glTranslatef(24f, 2f, 0f)
+        }
+        glColor4f(1f, 1f, 1f, 1f)
+
+        RenderHelper.disableStandardItemLighting()
+        glPopMatrix()
+    }
+
+    private fun getEnchantmentText(itemStack: ItemStack): TextComponent {
+        val textComponent = TextComponent()
+        val enchantmentList = EnchantmentUtils.getAllEnchantments(itemStack)
+        for (leveledEnchantment in enchantmentList) {
+            textComponent.add(leveledEnchantment.alias)
+            textComponent.addLine(leveledEnchantment.levelText, 0x9B90FF)
+        }
+        return textComponent
+    }
+
+    private fun getEnumHand(enumHandSide: EnumHandSide) =
+            if (mc.gameSettings.mainHand == enumHandSide && mainHand.value) EnumHand.MAIN_HAND
+            else if (mc.gameSettings.mainHand != enumHandSide && offhand.value) EnumHand.OFF_HAND
+            else null
+
+    private fun drawFrame(vertexHelper: VertexHelper, posBegin: Vec2d, posEnd: Vec2d) {
+        if (cornerRadius.value == 0f) {
+            if (filled.value)
+                RenderUtils2D.drawRectFilled(vertexHelper, posBegin, posEnd, ColorHolder(rFilled.value, gFilled.value, bFilled.value, aFilled.value))
+            if (outline.value && outlineWidth.value != 0f)
+                RenderUtils2D.drawRectOutline(vertexHelper, posBegin, posEnd, outlineWidth.value, ColorHolder(rOutline.value, gOutline.value, bOutline.value, aOutline.value))
+        } else {
+            if (filled.value)
+                RenderUtils2D.drawRoundedRectFilled(vertexHelper, posBegin, posEnd, cornerRadius.value.toDouble(), 8, ColorHolder(rFilled.value, gFilled.value, bFilled.value, aFilled.value))
+            if (outline.value && outlineWidth.value != 0f)
+                RenderUtils2D.drawRoundedRectOutline(vertexHelper, posBegin, posEnd, cornerRadius.value.toDouble(), 8, outlineWidth.value, ColorHolder(rOutline.value, gOutline.value, bOutline.value, aOutline.value))
+        }
+    }
+
+    override fun onUpdate() {
+        if (timer.tick(5L) && checkSetting()) MessageSendHelper.sendChatMessage("$chatName Totem pops is not yet implemented")
```

Comment:
don't add the setting if not added yet?

Reply:
Well just leave as as a TODO thing there lol. So someone sees it and they can impl it

Reply:
a comment in the code is fine. dead settings aren't fine